packages/devtools_app/test/flutter/debugger_screen_test.dart

```diff
@@ -75,7 +76,46 @@ void main() {
       expect(find.text('Console'), findsOneWidget);
 
       // test for stdio output
-      expect(find.text('test stdio'), findsOneWidget);
+      expect(
+          find.byWidgetPredicate((widget) =>
+              widget is RichText && widget.text.toPlainText() == 'test stdio'),
+          findsOneWidget);
+    });
+
+    testWidgets('Console area shows processed ansi text',
+        (WidgetTester tester) async {
+      when(debuggerController.stdio)
+          .thenReturn(ValueNotifier([_ansiCodesOutput()]));
+
+      await pumpDebuggerScreen(tester, debuggerController);
+
+      final finder = find.byWidgetPredicate((widget) =>
+          widget is RichText &&
+          widget.text.toPlainText() ==
+              'Ansi color codes processed for console');
+      expect(finder, findsOneWidget);
+      finder.evaluate().forEach((element) {
+        final richText = element.widget as RichText;
+        final textSpan = richText.text as TextSpan;
+        final secondSpan = textSpan.children[1] as TextSpan;
+        expect(
+          secondSpan.text,
+          'console',
+          reason: 'Text with ansi code should be in separate span',
+        );
+        expect(
```

Comment:
nit: you should be able to compare equality on the entire color rather than having to have a separate expect for red, green, and blue.